packages/invoices/src/InboundShipment/ListView/Toolbar.tsx

```diff
@@ -13,11 +13,11 @@ import { useDeleteSelectedInbounds, InboundRowFragment } from '../api';
 export const Toolbar: FC<{
   filter: FilterController;
 }> = ({ filter }) => {
-  const t = useTranslation();
+  const t = useTranslation('distribution');
```

Comment:
:nit inbound shipments are in `replenishment` and should be using translations from that file if they are specific.. or the translation should be in `common` if used in various places. 

I think this one should be in `common` as it will be used elsewhere. In which case you don't need to specify a namespace when calling `useTranslation` as it defaults to `common`

Reply:
Yeah, sure was wondering about that. There's actually an existing "Search by name" string in "inventory" as well -- should I consolidate them both into `common`?

Reply:
thanks - sounds good.
I do wonder about running some sort of usage check to determine which translation strings are unused 🤔
surely there's a script out there to do that..

Reply:
> should I consolidate them both into `common`?

Done 8a98121d

> I do wonder about running some sort of usage check to determine which translation strings are unused 🤔
> surely there's a script out there to do that..

Hmmmm... https://github.com/glebm/i18n-tasks

Will try that out later on.